kernel/src/Info.plist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>OSQuery</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.facebook.kext.OSQuery</string>
```

Comment:
I'd vote for `com.facebook.driver.osquery` or `com.facebook.iokit.osquery` (for IOKit-based extensions). But we could use `com.facebook.security.osquery` if we want to be enterprising.